src/tools/css/get-css.js

```diff
@@ -50,11 +50,13 @@ const getAvailableFeatures = (font) => {
 };
 
 const getVariableCSS = (font) => {
-	const name = slugify(font.summary["Font name"]);
 	let css = "";
+	let maxProps = 6;
+	const name = slugify(font.summary["Font name"]);
 	const fvar = font.get("fvar");
 	const variations = fvar ? fvar.instances : [];
 	for (const v in variations) {
+		let counter = 2;
```

Comment:
Could you name this variables more clearly? I guess this means how many properties are put on one line?